### ISSUE TITLE
Temporary Containers aren't temporary

### ISSUE THREAD

**Issue by mzguy** [Drive4ik/simple-tab-groups#607]
**Describe the bug**
Temporary containers are often (never?) deleted.

**To Reproduce**
Browse for a while by creating a group where tabs always open in temporary containers. Restart firefox, saving sessions, closing tabs, opening new ones, etc.

**Expected behavior**
The number of temporary containers matches the number of tabs that use them.

**Desktop (please complete the following information):**
 - OS and version: Windows 7
 - Firefox version: 76.0.1
 - Simple Tab Groups version: 4.5.1

**Comment by Drive4ik** [comment]
STG delete unused temporary containers only after restart browser. I changed the behavior to this, because it happens that you accidentally close a tab with a temporary container, and the addon has already deleted this temporary container, and it is impossible to restore a closed tab in the same container.
You can manually reopen the tabs with temporary containers
![изображение](https://user-images.githubusercontent.com/7843031/81944191-12a49400-9605-11ea-9069-fc165c675b4e.png)

**Comment by mzguy** [comment]
That's what I was saying in my problem description. Restarting Firefox didn't clear them. I recently installed Multi-Account Containers and enabled container sync. Could that be related to the issue?

**Comment by mzguy** [comment]
Many of the names (Like Temporary Container 249) are duplicated many times. If I ignore the duplicates, the count approximately matches the number of discarded by not deleted tabs I have that use temporary containers (I figure these persist past browser restart). However, with duplicates I have least 30 more than must be necessary.

**Comment by Drive4ik** [comment]
Can you close all the tabs that use temporary containers and restart FF? If some temporary containers are saved (but this should not be, I just checked it), then the name of these containers has changed. That is, if there is a `Temporary Container 249` container then its ID will be `firefox-container-249`. In this case, the container is temporary. But the user can change the name of the container to another - and then this container ceases to be temporary (in both cases a notification will be displayed)
If you still have containers that you consider duplicates, delete them.

**Comment by mzguy** [comment]
I see now. Yes, the culprit is the Multi-Account Containers extension. It's creating tons of duplicates of temporary containers. If I disable the extension the problem goes away. I suspect it's the container sync function of that extension. I'd like to be able to use that extension if possible, so if you have a way to be compatible that would be helpful.

**Comment by Drive4ik** [comment]
I will not write an analog Multi-Account Containers, but I can try to write a plugin for STG, to synchronize containers, of course excluding temporary containers. But I don’t know if it makes sense to write it? How many users will need it?

**Comment by mzguy** [comment]
It would be most useful to synchronize not just containers, but also groups! After all, if the user is synchronizing tabs, it would make sense to also synchronize their organization. Synchronizing both groups and containers would provide a truly seamless experience, and I am sure many users would love it.

**Comment by Drive4ik** [comment]
This is not yet possible ... 
https://github.com/Drive4ik/simple-tab-groups/issues/291#issuecomment-451531973

**Comment by mzguy** [comment]
In that case I think it makes sense to wait until you can implement full groups sync with all settings, and include containers at that time.

**Comment by Drive4ik** [comment]
Actually, I do it)
